build.js

```diff
@@ -397,6 +399,25 @@ function handle(options) {
 }
 
 function dataToHtml(skeleton, rawBrowsers, tests, compiler) {
+  var wrapWithScript = function(assertions) {
```

Comment:
This function can be hoisted out of dataToHtml

Reply:
I agree that it *could*, but there's another five functions that exist below it within the `dataToHtml` closure that don't need to be there, either. I was aiming for consistency. Should I move all of them in a separate refactor, perhaps?

Reply:
I suppose that's fine too.